tests/console/bind.pm

```diff
@@ -0,0 +1,76 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: bind upstream testsuite
+#          prepare, build, fix broken tests and execute testsuite
+# Maintainer: Jozef Pupava <jpupava@suse.com>
+
+use base 'consoletest';
+use testapi;
+use strict;
+use utils 'zypper_call';
+use version_utils 'is_sle';
+
+sub run {
+    select_console 'root-console';
+
+    if (is_sle('=12-SP3')) {
+        # SLE 12-SP3 neeed gpg-offline to build
+        assert_script_run 'wget http://download.suse.de/ibs/SUSE:/SLE-12:/GA/standard/noarch/gpg-offline-0.1-10.4.noarch.rpm';
```

Comment:
What's about putting it as a variable, so no code change is required if we get package update or new path?

Reply:
Yes, I'm also not happy about this hard coded links and packages, but it is needed. When It will change then I will change the code and restart the test, I think it will be easier than modifying variable and cloning job to apply the change. I would leave it for now, I will have to change it soon for 12SP4 and 15SP1